### PR TITLE
refactor: improve UI of new login page

### PR DESCRIPTION
**Pull request by ArshdeepGrover**
Screenshot from 2022-12-29 12-11-20
Screenshot from 2022-12-29 12-11-48
Screenshot from 2022-12-29 12-12-01

### CODE REVIEW

File: apps/commudle-admin/src/app/components/login/login.component.scss [hunk 1]
```diff
@@ -16,5 +16,8 @@
       text-align: center;
       text-transform: uppercase;
     }
+    button {
+      @apply com-mt-8 com-mb-20 #{!important};
```
Comment (Ashwin-op) [inline apps/commudle-admin/src/app/components/login/login.component.scss:20]: Let's avoid !important

File: apps/commudle-admin/src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -58,6 +59,11 @@ export class LoginComponent implements OnInit, OnDestroy {
         }
       }),
     );
+    this.libAuthWatchService.currentUser$.subscribe((currentUser) => {
```
Comment (Ashwin-op) [inline apps/commudle-admin/src/app/components/login/login.component.ts]: Add to subscriptions array

**Review by Ashwin-op — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/components/login/login.component.ts [hunk 3]
```diff
@@ -78,7 +71,9 @@ export class LoginComponent implements OnInit, OnDestroy {
   }
 
   redirect(): void {
-    window.location.href = window.location.origin + this.activatedRoute.snapshot.queryParams.redirect || '/';
+    window.location.href = this.activatedRoute.snapshot.queryParams.redirect
+      ? window.location.origin + this.activatedRoute.snapshot.queryParams.redirect
```
Comment (Ashwin-op) [inline apps/commudle-admin/src/app/components/login/login.component.ts:75]: Why was this needed?

**Comment by ArshdeepGrover** [comment]
Need some major changes so closed and make new PR

**Pull request closed without merging**